tbx/static_src/sass/components/_grid.scss

```diff
@@ -101,7 +101,7 @@
     }
 
     &__work-image {
-        grid-column: 2 / span 4;
+        grid-column: 1 / span 5;
```

Comment:
This now leaves the caption stranded so we'll need to look at getting that styled separately.

Reply:
Making the img tag 100% width solves it. Applied in the grid css to avoid having to create a new component just for this.